packages/jest-transform/src/shouldInstrument.ts

```diff
@@ -59,10 +73,9 @@ export default function shouldInstrument(
     // still cover if `only` is specified
     !options.collectCoverageOnlyFrom &&
     options.collectCoverageFrom.length &&
-    micromatch(
-      [replacePathSepForGlob(path.relative(config.rootDir, filename))],
-      options.collectCoverageFrom,
-    ).length === 0
+    !globsToMatcher(options.collectCoverageFrom)(
```

Comment:
I just noticed that this will end up using the `dot: true` option where before that wasn't used here (and above in this file). This seems like it should be fine, but I don't really have enough context here to know for sure.

Reply:
yeah, seems fine. HasteMap should filter out any dotfiles we don't want already (I _think_)